torchtitan/experiments/deepseek_v3/model.py

```diff
@@ -472,25 +468,22 @@ class MoE(nn.Module):
     # 1. "torch_all_to_all"
     # 2. "symm_mem" (see `setup_symm_mem` below)
     shuffle_method = "torch_all_to_all"
-    # Group GEMM method
-    group_mm = "torch"  # "  # [ torch, torchao, ds (ds==fp8) ]
-
-    if group_mm == "ds" and not DEEPGEMM_AVAILABLE:
-        print(
-            "DeepGemm is not available on your system.  Please install and try again."
-        )
-        raise ValueError("Missing DeepGemm install")
 
     # Symmetric memory buffers shared by all MoE instances across layers
     token_send_buf: Optional[torch.Tensor] = None
     token_gather_buf: Optional[torch.Tensor] = None
 
-    def __init__(self, config):
+    def __init__(self, config, custom_activation=None):
```

Comment:
nit: no need to add `custom_activation` argument here, we should just use `MLP.act_fn` to initialize the strategies.

Reply:
updated to use MLP.act_fn